java/src/lib.rs

```diff
@@ -89,10 +79,59 @@ fn redis_value_to_java<'local>(
             data: _,
             attributes: _,
         } => todo!(),
-        Value::Push { kind: _, data: _ } => todo!(),
+        // Create a java `Map<String, Object>` with two keys:
+        //   - "kind" which corresponds to the push type, stored as a `String`
+        //   - "values" which corresponds to the array of values received, stored as `Object[]`
+        // Only string messages are supported now by Redis and `redis-rs`.
+        Value::Push { kind, data } => {
+            let hash_map = env.new_object("java/util/HashMap", "()V", &[])?;
+
+            let kind_str = env.new_string("kind")?;
+            let kind_value_str = env.new_string(format!("{kind:?}"))?;
+
+            let _ = env.call_method(
```

Comment:
I don't think we need the `let _ =` here if we are using `?` to handle the Result. The Map case doesn't use it so this shouldn't need it either.